test-runner/adapters/direct_python_sdk/direct_module_api.py

```diff
@@ -8,48 +8,49 @@
 import json
 from ..print_message import print_message
 from ..abstract_module_api import AbstractModuleApi
-from azure.iot.hub.devicesdk import ModuleClient
+from azure.iot.hub.devicesdk.sync_clients import ModuleClient as ModuleClientSync
```

Comment:
>.sync_clients [](start = 28, length = 13)

No, do not import from this module. Use:

from azure.iot.hub.devicesdk import ModuleClient

1) Inner module structure is subject to change. Import from the package to ensure your code is protected
2) Why rename it as ModuleClientSync, when there's no alternative? In the synchronous direct_module_api the module client is implied synchronous.